.github/workflows/release.yml

```diff
@@ -24,29 +31,29 @@ jobs:
             const version = require(`${process.env.GITHUB_WORKSPACE}/package.json`).version;
             core.setOutput('repo', context.repo.repo);
             core.setOutput('version', version);
-            const time = new Date(Date.now()).toISOString().slice(0,10);
-            const titleLink = version.replace(/\./g,'') + '-' + time;
-            core.setOutput('titleLink', titleLink);
-            try {
-              var latestRelease = await github.repos.getLatestRelease({
-                owner: context.repo.owner,
-                repo: context.repo.repo
-              });
-              const latestTag = latestRelease.data.tag_name;
-              core.setOutput('commitLink', `compare/${latestTag}...v${version}`);
-            } catch (error) {
-              if (error.message === 'Not Found') {
-                core.setOutput('commitLink', `commits/v${version}`);
-              } else {
-                core.setFailed(error.message);
-              }
-            }
-            return true;
 
-      - name: Generate Package
+      - name: Push commit
         run: |
-          yarn install
-          yarn run build
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git add .
+          git commit -m "chore(release): :bookmark: release v${{steps.version.outputs.version}}"
+          git push https://vivaxy:${{secrets.GITHUB_TOEKN}}@github.com/vivaxy/vscode-conventional-commits.git
```

Comment:
Should I add my `GITHUB_TOEKN` into the repo setting?
It seems we are pushing commit as `vivaxy`, but why `user.name` and `user.email` is set to be `github-actions[bot]`?

Reply:
1. This secret is provided by GitHub action, so you do not need anymore things to do. The permission of `secrets.GITHUB_TOKEN` is very limitation, and it is restricted to be used in workflows, and does not have the permission to obtain special data in the repository or the data in other repositories of the owner. (Reference [authentication-in-a-workflow](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)). 
2. I think that the commit is edited by CI, so I use the `github-actions[bot]` to commit, then the push commit is just the commit edited before. I l learn it from [authenticate-with-github-using-a-token](https://stackoverflow.com/questions/18935539/authenticate-with-github-using-a-token). And the history of this repo will only add a commit edited by `github-actions[bot]`. Most actions that have committed are used this way, such as [TriPSs/conventional-changelog-action](https://github.com/TriPSs/conventional-changelog-action/blob/b40e4b899fd594aab7042c017baad190e3f4b5ca/src/helpers/git.js#L40). I think the name is not useful, I will change it later and set it as same as the example action.

Reply:
Edit in https://github.com/vivaxy/vscode-conventional-commits/pull/61/commits/ce3c6f474d0272ae396bafb2653ecb935b0f4a6a.